Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -1310,8 +1322,1295 @@ double approximate_symmetric_Hausdorff_distance(const TriangleMesh& tm1,
+      // for checking if two meshes are close.
+      const double dist = bounded_error_Hausdorff_impl<Kernel>(
+        tm1, tm2, error_bound, vpm1, vpm2,
+        infinity_value, initial_bound, -FT(1),
+        tm1_tree, tm2_tree, stub);
+      if (dist > hdist) hdist = dist;
+    }
+    if (hdist > distance) distance = hdist;
+    timer.stop();
+    // std::cout << "* time operator() computation (sec.): " << timer.time() << std::endl;
+  }
+
+  void join(Bounded_error_distance_computation& rhs) {
+    distance = (CGAL::max)(rhs.distance, distance);
+  }
+};
+
+#endif // defined(CGAL_LINKED_WITH_TBB) && defined(CGAL_METIS_ENABLED)
+
+template< class Concurrency_tag,
+          class Kernel,
+          class TriangleMesh1,
+          class TriangleMesh2,
+          class VPM1,
+          class VPM2,
+          class NamedParameters1,
+          class NamedParameters2,
+          class OutputIterator >
+double bounded_error_one_sided_Hausdorff_impl(
+  const TriangleMesh1& tm1,
+  const TriangleMesh2& tm2,
+  const typename Kernel::FT error_bound,
+  const typename Kernel::FT distance_bound,
+  const bool compare_meshes,
+  const VPM1& vpm1,
+  const VPM2& vpm2,
+  const NamedParameters1& np1,
+  const NamedParameters2& np2,
+  OutputIterator& out)
+{
+  #if !defined(CGAL_LINKED_WITH_TBB) || !defined(CGAL_METIS_ENABLED)
+  CGAL_static_assertion_msg(
+    !(boost::is_convertible<Concurrency_tag, CGAL::Parallel_tag>::value),
+    "Parallel_tag is enabled but at least TBB or METIS is unavailable.");
+  #endif
+
+  using FT = typename Kernel::FT;
+
+  using TM1 = TriangleMesh1;
+  using TM2 = TriangleMesh2;
+
+  using TM1_primitive = AABB_face_graph_triangle_primitive<TM1, VPM1>;
+  using TM2_primitive = AABB_face_graph_triangle_primitive<TM2, VPM2>;
+
+  using TM1_traits = AABB_traits<Kernel, TM1_primitive>;
+  using TM2_traits = AABB_traits<Kernel, TM2_primitive>;
+
+  using TM1_tree = AABB_tree<TM1_traits>;
+  using TM2_tree = AABB_tree<TM2_traits>;
+
+  using Face_handle_1 = typename boost::graph_traits<TM1>::face_descriptor;
+  using Face_handle_2 = typename boost::graph_traits<TM2>::face_descriptor;
+
+  using Timer = CGAL::Real_timer;
+
+  #if defined(CGAL_LINKED_WITH_TBB) && defined(CGAL_METIS_ENABLED) && defined(USE_PARALLEL_BEHD)
+  using Point_3       = typename Kernel::Point_3;
+  using TMF           = CGAL::Face_filtered_graph<TM1>;
+  using TMF_primitive = AABB_face_graph_triangle_primitive<TMF, VPM1>;
+  using TMF_traits    = AABB_traits<Kernel, TMF_primitive>;
+  using TMF_tree      = AABB_tree<TMF_traits>;
+  using TM1_wrapper   = Triangle_mesh_wrapper<TMF, VPM1, TMF_tree>;
+  using TM2_wrapper   = Triangle_mesh_wrapper<TM2, VPM2, TM2_tree>;
+
+  std::vector<TMF> tm1_parts;
+  std::vector<TMF_tree> tm1_trees;
+  std::vector<boost::any> tm_wrappers;
+  #endif // defined(CGAL_LINKED_WITH_TBB) && defined(CGAL_METIS_ENABLED)
+
+  Timer timer;
+  std::cout.precision(20);
```

Comment:
this should be hidden too (17 is enough btw)

Reply:
Done.